plugins/frontend/backstage-plugin-security-insights/src/components/DependabotAlertsTable/DependabotAlertsTable.test.tsx

```diff
@@ -76,7 +76,6 @@ describe('Dependabot alerts overview', () => {
   setupRequestMockHandlers(worker);
 
   beforeEach(() => {
-    worker.resetHandlers();
```

Comment:
Was this one the reason for failing tests?

Reply:
No, I just did some cleaning becuase the setupRequestMockHandlers actually configures the lifecycle functions for it, so this seemed like a duplicate here


```ts
/**
 * Sets up handlers for request mocking
 * @public
 * @param worker - service worker
 */
export function setupRequestMockHandlers(worker: {
  listen: (t: any) => void;
  close: () => void;
  resetHandlers: () => void;
}) {
  beforeAll(() => worker.listen({ onUnhandledRequest: 'error' }));
  afterAll(() => worker.close());
  afterEach(() => worker.resetHandlers());
}
```